project/Dependencies.scala

```diff
@@ -9,15 +9,15 @@ object Dependencies {
   val automationGoogleV = "1.30.5"
   val scalaLoggingV = "3.9.5"
   val scalaTestV = "3.2.17"
-  val http4sVersion = "1.0.0-M38" //Do not upgrade to M40 until workbench-libs does (missing M40 for blaze as of 08/2023
+  val http4sVersion = "1.0.0-M40" //Do not upgrade to M40 until workbench-libs does (missing M40 for blaze as of 08/2023
```

Comment:
out of curiosity, are we using any of the features in 1.0.0-MXX branch?, and if not, does it make sense to go with their officially released version of [v0.23.24](https://github.com/http4s/http4s/releases/tag/v0.23.24)?

Reply:
unfortunately, 1.0.0 is not binary compatible with 0.23 series...

Reply:
The issue is finding all the packages that have the updated cats-effect (3.5.+)

Blaze, a subproject of http4s, did not support the cats-effect change until recently (even though we had upgrade our cats-effect 2 months ago). 

Blaze can work against 1.0.0-M39, but I couldn't find one for 0.23.23 or 0.23.24.

But with 1.0.0-M40, metrics.prometheus is not supported. 

So now I am back trying 0.23.23 and trying to find compatible Blaze and metrics.prometheus. 

At this point it might be better to back out cats-effect until all our packages that use it are upgraded to support the major change in 3.5.